### PR TITLE
add new task, alg4

### PR DESCRIPTION
**Pull request by marina-nazarenko**

### CODE REVIEW

File: src/task1/Alg4.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import java.util.Arrays;
+
+public class Alg4 {
+    public static void main(String[] args) {
+        Alg4 alg4 = new Alg4();
+        alg4.replacement();
+
+    }
+
+    private void replacement() {
```
Comment (denis-tumel) [inline src/task1/Alg4.java:12]: rename method name

File: src/task1/Alg4.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        for (int i = 0; i < a.length; i++) {
+            System.out.print(a[i] + " ");
+        }
+
+    }
+}
+
+
+
+
```
Comment (denis-tumel) [inline src/task1/Alg4.java:39]: redurant lines

File: src/task1/Alg4.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    public static void main(String[] args) {
+        Alg4 alg4 = new Alg4();
+        alg4.replacement();
+
+    }
+
+    private void replacement() {
+        int[] a = new int[]{1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+        int max = a[0];
+        int imax = 0;
```
Comment (denis-tumel) [inline src/task1/Alg4.java:15]: iMax

File: src/task1/Alg4.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        int min = a[0];
+        int imin = 0;
```
Comment (denis-tumel) [inline src/task1/Alg4.java:17]: iMin

File: src/task1/Alg4.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
+package task1;
+
+import java.util.Arrays;
+
+public class Alg4 {
+    public static void main(String[] args) {
+        Alg4 alg4 = new Alg4();
+        alg4.replacement();
+
```
Comment (denis-tumel) [inline src/task1/Alg4.java:9]: redurant line

File: src/task1/Alg4.java (hunk 1 above)
Comment (marina-nazarenko) [reply src/task1/Alg4.java:12]: fixed

File: src/task1/Alg4.java (hunk 2 above)
Comment (marina-nazarenko) [reply src/task1/Alg4.java:39]: fixed

File: src/task1/Alg4.java (hunk 3 above)
Comment (marina-nazarenko) [reply src/task1/Alg4.java:15]: fixed

File: src/task1/Alg4.java (hunk 4 above)
Comment (marina-nazarenko) [reply src/task1/Alg4.java:17]: fixed

File: src/task1/Alg4.java (hunk 5 above)
Comment (marina-nazarenko) [reply src/task1/Alg4.java:9]: fixed

**Review by denis-tumel — APPROVED** [review]

**Pull request merged**